userguide/intro.rst

```diff
@@ -117,8 +117,15 @@ These major features are new in this version:
   :menuselection:`System --> SSH Keypairs`
   screens.
 
+* :ref:`Configuring reporting graphs <Configure Reporting>` has been
+  added with a new menu option added to
```

Comment:
The two instances of "added" are confusing.  Is it clearer to say
`` "A new menu option in :menuselection:... allows :ref:`configuring reporting graphs <Configure Reporting>`." ``